acinclude.m4

```diff
@@ -384,7 +384,7 @@ dnl like WX_ARG_WITH but uses AC_ARG_ENABLE instead of AC_ARG_WITH
 dnl usage: WX_ARG_ENABLE(option, helpmessage, var, [enablestring], [default])
 dnl
 dnl enablestring can be omitted or a literal string "disable" and allows to
-dnl show "checking for --disable-foo" message when running configure instead of
+dnl show "checking for --disable-foo" message when running configure.acstead of
```

Comment:
Ugh, sorry, but you really, really shouldn't be running automatic replacements in the entire search tree and if you do it, you should at least be using the right pattern, with `.` metacharacter being escaped.

Reply:
Oops, sorry! I did quickly review the 54 something instances but missed that one.  I've fixed that typo, tightened the regexp and pushed, after validating that it seems to configure and build fine on my end.